packages/api/tests/animal_group.test.js

```diff
@@ -172,15 +172,19 @@ describe('Animal Group Tests', () => {
         expect(res.status).toBe(200);
         // Should return first two animal groups
         expect(res.body.length).toBe(2);
+        // Sometimes response is returned in different order than created
+        const idMap = res.body.map((group) => groups.findIndex(({ name }) => name === group.name));
```

Comment:
I think naming an array `idMap` is misleading.

Reply:
I added a todo since it is just a patch for flaky test. Is the name any better?

Reply:
Yes! The new name sounds perfect ❤️ Thank you!
I think your solution is good. Another solution maybe to find a corresponding group from `groups` in `forEach`.